sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -386,7 +386,7 @@ def body_as_json(self, encoding="UTF-8"):
 
         :param encoding: The encoding to use for decoding event data.
          Default is 'UTF-8'
-        :rtype: dict
+        :rtype: Dict[str, Any]
```

Comment:
quick question: have you run sphinx to see how it's rendered in the sphinx doc?

I'm wondering how it would look as `rtype` is a sphinx thing while `Dict` is from typing -- Laurent mentioned in the [issue](https://github.com/Azure/azure-sdk-for-python/issues/22051): "Sphinx do not use typing, but a more classical system"

Reply:
I have not, but there are actually places in the docs where we already have typing.Dict and it looks fine. For example, here's Dict looks in the APIView for type `properties` in `EventData`: https://apiview.dev/Assemblies/Review/3e49ce1ca870470289360023dcb51edf#

Reply:
okay.. the doc of "<xref:Iterable>[<xref:Dict>[<xref:str,Any>]]" doesn't seem correct -- which means changing from `rtype: dict` to `:rtype: Dict[str, Any]` would lead to Api reference doc "broken" which is not a good documentation experience.

I think we should keep the `rtype: dict` or changed to `rtype: dict[str, any]` instead of using typing one `:rtype: Dict[str, Any]`.

Reply:
sounds good - does this mean we should update all Dict to dict then?

Reply:
as per OOB discussion: leaving this as Dict for now. Plan on updating all types that don't follow sphinx rules in the future.